tests/unit/utils/test_version.py

```diff
@@ -969,6 +971,8 @@ class FakeWebEngineSettings:
         monkeypatch.delattr(sys, 'frozen', raising=False)
 
     substitutions['uptime'] = "1:23:45"
+    substitutions['autoconfig_loaded'] = (
+        "yes" if autoconfig_loaded else "no")
```

Comment:
I'm not sure why `'uptime'` gets added here (instead of adding it in the `substitutions = { ... }` block above). Can you please move both to the place above?

Reply:
I think I overloked the `substitutions = { ... }` block, I will move it